projects/q-acc/index.js

```diff
@@ -0,0 +1,41 @@
+const ADDRESSES = require('../helper/coreAssets.json')
+const abi = require('./abi.json')
+const { config } = require('./config')
+
+async function tvl(api) {
+    const collateralSupplies = await api.multiCall({
+        abi: abi.getVirtualCollateralSupply,
+        calls: Object.values(config).map(c => ({
+            target: c.bondingCurve,
+        })),
+    })
+
+    const sumOfCollateralSupplies = collateralSupplies.reduce((acc, curr) => acc + BigInt(curr), BigInt(0))
+
+    const issuanceSuppliesFromCurves = await api.multiCall({
+        abi: abi.getVirtualIssuanceSupply,
+        calls: Object.values(config).map(c => ({
+            target: c.bondingCurve,
+        })),
+    })
+
+    const issuanceSupplies = Object.values(config).map((configItem, index) => ({
```

Comment:
If we're counting the POL used to purchase ABC tokens then we shouldn't also count the ABC supply